tower-batch/src/lib.rs

```diff
@@ -7,5 +7,16 @@ mod worker;
 
 type BoxError = Box<dyn std::error::Error + Send + Sync + 'static>;
 
-pub use self::layer::BufferLayer;
-pub use self::service::Buffer;
+pub enum BatchControl<R> {
```

Comment:
I called this `BatchControl` rather than `BatchRequest` to emphasize that it's a control channel for the batchable service.